### ISSUE TITLE
Set "Paste as plain text" by default

### ISSUE THREAD

**Issue by chevi** [taqueci/redmine_wysiwyg_editor#22]
Hi

Could you please set "Paste as plain text" as default?

Steps to reproduce:

1. open github (for example: https://github.com/taqueci/redmine_wysiwyg_editor/issues/21) and copy text from that page as on picture:
![image](https://user-images.githubusercontent.com/1729590/45954540-f742da80-c015-11e8-86e9-6ba7e5e3b63f.png)
2. Paste text into editor
![image](https://user-images.githubusercontent.com/1729590/45954579-0e81c800-c016-11e8-81e2-9b470244fb5d.png)
3. Switch to Preview or Save note

Result:
![image](https://user-images.githubusercontent.com/1729590/45954592-19d4f380-c016-11e8-979e-1df7b86352a9.png)
![image](https://user-images.githubusercontent.com/1729590/45954688-54d72700-c016-11e8-8c26-271d70ef91d3.png)

I think that plain text should be better
https://stackoverflow.com/a/17421926

**Comment by taqueci** [comment]
Hi @chevi 

I can not do it because some users like pasting as HTML.
I know more improvements are required.

I have implemented the functionality to configure the editor. (#24)
You can set plain text pasting as default by using [View customize plugin](https://github.com/onozaty/redmine-view-customize).
Please try to install the plugin and set with:
```js
var WYSIWYG_EDITOR_SETTING = {
    plugins: 'link lists hr table textcolor paste',
    paste_as_text: true
};
```

**Comment by taqueci** [comment]
Paste plugin is enabled in version 0.3.4.
For removing extra attributes, please use the clear formatting button implemented by #21.
